android/app/src/main/java/com/pccontrol/voice/services/WebSocketManager.kt

```
@@ -143,7 +146,17 @@ class WebSocketManager private constructor(private val context: Context) {
     /**
      * Send audio data for transcription.
      */
-    suspend fun sendAudioData(audioData: ByteArray, mimeType: String = "audio/webm"): Result<Unit> {
+    fun sendAudioData(audioData: ByteArray) {
+        scope.launch {
+            try {
+                webSocketClient?.sendAudioData(audioData, "audio/webm")
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)